src/js/netjsongraph.render.js

```diff
@@ -574,9 +579,20 @@ class NetJSONGraphRender {
       self.config.clustering &&
       self.config.clusteringThreshold < JSONData.nodes.length
     ) {
+      // Initial clustering state should match the current zoom level
+      const initialClusteringState =
+        self.leaflet.getZoom() < self.config.disableClusteringAtLevel;
+
       let {clusters, nonClusterNodes, nonClusterLinks} =
         self.utils.makeCluster(self);
 
+      // Only show clusters if we're below the disableClusteringAtLevel
+      if (!initialClusteringState) {
```

Comment:
Wouldn't this obtain the same result without the need of the additional lines above?
```suggestion
      if (!self.leaflet.getZoom() < self.config.disableClusteringAtLevel) {
```

It makes sense to store computations in variabeles only if we reuse them, otherwise it's just a waste of space in the library.

Reply:
You're right I also thought it could be useful for reusability but right now it's a waste of space.

I'll change the logic and remove the variable